src/D2L.Bmx/WriteHandler.cs

```diff
@@ -85,6 +85,27 @@ bool useCredentialProcess
 				+ $" --role {awsCredsInfo.Role}"
 				+ $" --duration {awsCredsInfo.Duration}";
 		} else {
+			if( File.Exists( SharedCredentialsFile.DefaultConfigFilePath ) ) {
+				string profileName = $"profile {profile}";
+				var defaultConfigFile = parser.ReadFile( SharedCredentialsFile.DefaultConfigFilePath );
+				if( defaultConfigFile.Sections.ContainsSection( profileName )
+					&& defaultConfigFile[profileName].ContainsKey( "credential_process" ) ) {
+
+					if( defaultConfigFile[profileName].Count == 1 ) {
+						defaultConfigFile.Sections.RemoveSection( profileName );
+					} else {
+						defaultConfigFile[profileName].RemoveKey( "credential_process" );
+					}
+					parser.WriteFile( SharedCredentialsFile.DefaultConfigFilePath, defaultConfigFile );
+					Console.WriteLine(
+						"""
+						An existing profile with the same name using the `credential_process` key was found in the default config file.
+						The key has been removed, and static credentials will be used for the profile.
+						To continue using non-static credentials, rerun the command with the --use-credential-process flag.
```

Comment:
"key" is a pretty technical term. I find it a little weird/confusing for user-facing language.
Maybe use "setting", as that's what the AWS doc uses:
https://docs.aws.amazon.com/sdkref/latest/guide/feature-process-credentials.html

```suggestion
						An existing profile with the same name using the `credential_process` setting was found in the default config file.
						The setting has been removed, and static credentials will be used for the profile.
						To continue using non-static credentials, rerun the command with the --use-credential-process flag.
```

Reply:
Yea I went back and forth on what to call that but setting does sound better